cita-chain/core/src/libchain/chain.rs

```diff
@@ -253,8 +238,15 @@ impl Config {
 }
 
 impl BloomGroupDatabase for Chain {
-    fn blooms_at(&self, _position: &BloomGroupPosition) -> Option<BloomGroup> {
-        unimplemented!()
+    fn blooms_at(&self, position: &BloomGroupPosition) -> Option<BloomGroup> {
+        let p = LogGroupPosition::from(position.clone());
```

Comment:
Why not use `let position = LogGroupPosition::from(position.clone());`

Reply:
There is a local variable already named `position`.  In my view, an abbreviated variable name is acceptable while expliciting variable type.

Reply:
> There is a local variable already named position.

yeah, i see it. that's ok for this function.

> variable bindings can be shadowed.

> an abbreviated variable name is acceptable

I don't agree. but that's ok for me. Forget it.